### ISSUE TITLE
Importing griddly prints an empty line

### ISSUE THREAD

**Issue by cswinter** [Bam4d/Griddly#177]
Importing griddly prints an empty line:

```
⋊> ~/s/enn-incubator on clemens/hyperstate ⨯ pip list | grep griddly                                                                                                                                                                                                     (base) 08:56:04
griddly             1.2.36
⋊> ~/s/enn-incubator on clemens/hyperstate ⨯ python                                                                                                                                                                                                                      (base) 08:56:08
Python 3.7.3 (default, Mar 27 2019, 22:11:17)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import griddly

>>>
```

This causes a lot of empty lines to get printed in https://github.com/entity-neural-network/incubator when spawning multiple processes that all import griddly.

**Comment by Bam4d** [comment]
:D weird! Ill try fix it in next version

**Comment by Bam4d** [comment]
I think this actually comes from the openai gym dependency: 
![image](https://user-images.githubusercontent.com/1370765/157390959-1bff45a9-bd80-4795-893d-1af6923a13a2.png)

**Comment by cswinter** [comment]
Good find, it's probably this: https://github.com/openai/gym/issues/2632

It's fixed as of 0.23:
```
⋊> ~ python                                                                                                                                                                                                                                                              (base) 08:15:37
Python 3.7.3 (default, Mar 27 2019, 22:11:17)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gym
>>>
⋊> ~ pip list | grep gym                                                                                                                                                                                                                                                 (base) 08:15:49
gym                 0.23.0
gym-notices         0.0.4
```
